server/templates/home.html

```diff
@@ -1,67 +1,180 @@
+                                    <span class="glyphicon glyphicon-pencil" data-toggle="tooltip"
+                                          title="Edit the sample name"></span>
+                                </span>
+                        </div>
+
                     </div>
                 </form>
 
-                <div class="resumable-drop">
-                    2. Drop files here or <a class="resumable-browse btn btn-default">Browse your computer</a>
+                <div class="resumable-drop col-sm-12 col-xs-12">
+                    Drop files here or <a class="resumable-browse btn btn-default">Browse your computer</a>
                 </div>
+            </div>
 
-                <div class="current-uploads">
-                    <h2>Current uploads:</h2>
-                    <span class="all-transfer-controls">
-                        <button class="progress-resume-link btn btn-success">Resume all</button>
-                        <button class="progress-pause-link btn btn-warning">Pause all</button>
-                    </span>
-                    <ul id="resumable-transfers">
-                        <li id="resumable-progress-template" class="resumable-progress row">
-                            <p class="progress-filename col-xs-5"></p>
-                            <div class="progress-wrapper col-xs-7">
-                                <div class="progress-controls">
-                                    <button class="progress-cancel-link btn btn-danger"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Cancel</button>
-                                    <button class="progress-retry-link btn btn-success"><span class="glyphicon glyphicon-refresh" aria-hidden="true"></span> Retry</button>
-                                </div>
-
-                                <div class="progress">
-                                    <div class="progress-bar progress-bar-striped active" role="progressbar"></div>
-                                </div>
-                            </div>
-                        </li>
-                    </ul>
-                </div>
+            <table class="table table-condensed table-data">
+                <thead>
+                <tr class="active">
+                    <th>Sample Name</th>
+                    <th>Files</th>
+                    <th>Meta Data</th>
+                    <th>Status</th>
+                    <th>Options</th>
+                </tr>
+                </thead>
+                <tbody class="sample-template">
+                <tr class="active sample-header-template">
+                    <td class="sample-name"></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td class="sample-option">
+                        <span class="option-icon resumable-add add-files glyphicon glyphicon-plus" data-toggle="tooltip"
+                              title="Add file(s) to sample"></span> &thinsp;
```

Comment:
Rather than using space characters to separate these, you should adjust the spacing using the padding (or margin) within css.